client-v2/src/components/FrontsEdit/FrontSection.tsx

```diff
@@ -230,15 +232,16 @@ class FrontSection extends React.Component<
                   >
                     {isHidden ? 'Unhide' : 'Hide'}
                   </FrontHeaderButton>
-                  <FrontHeaderButton
-                    data-testid="rename-front-button"
-                    onClick={this.renameFront}
-                    size="l"
-                  >
-                    Rename
-                  </FrontHeaderButton>
                 </>
               )}
+              <FrontHeaderButton
```

Comment:
Nit: It would be more idiomatic to style in the component -- in this case at the point where FrontHeaderButton is created on L90.